diff --git a/window.c b/window.c
index d65bf50..8757b26 100644
--- a/window.c
+++ b/window.c
@@ -1187,7 +1187,7 @@ window_pane_find_up(struct window_pane *wp)
 
 	edge = wp->yoff;
 	if (edge == 0)
-		edge = wp->window->sy + 1;
+		return wp;
 
 	left = wp->xoff;
 	right = wp->xoff + wp->sx;
@@ -1230,7 +1230,7 @@ window_pane_find_down(struct window_pane *wp)
 
 	edge = wp->yoff + wp->sy + 1;
 	if (edge >= wp->window->sy)
-		edge = 0;
+		return wp;
 
 	left = wp->xoff;
 	right = wp->xoff + wp->sx;
@@ -1273,7 +1273,7 @@ window_pane_find_left(struct window_pane *wp)
 
 	edge = wp->xoff;
 	if (edge == 0)
-		edge = wp->window->sx + 1;
+		return wp;
 
 	top = wp->yoff;
 	bottom = wp->yoff + wp->sy;
@@ -1316,7 +1316,7 @@ window_pane_find_right(struct window_pane *wp)
 
 	edge = wp->xoff + wp->sx + 1;
 	if (edge >= wp->window->sx)
-		edge = 0;
+		return wp;
 
 	top = wp->yoff;
 	bottom = wp->yoff + wp->sy;
