Description: Elide some documentation regarding Sarge, long obsolete
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Forwarded: no
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-03-06

Index: rxvt-unicode/doc/rxvt.7.pod
===================================================================
--- rxvt-unicode.orig/doc/rxvt.7.pod	2013-12-19 10:26:34.871521712 -0500
+++ rxvt-unicode/doc/rxvt.7.pod	2013-12-19 10:26:34.871521712 -0500
@@ -546,12 +546,9 @@
 
 =head3 My numerical keypad acts weird and generates differing output?
 
-Some Debian GNUL/Linux users seem to have this problem, although no
-specific details were reported so far. It is possible that this is caused
-by the wrong C<TERM> setting, although the details of whether and how
-this can happen are unknown, as C<TERM=rxvt> should offer a compatible
-keymap. See the answer to the previous question, and please report if that
-helped.
+Some Debian GNU/Linux users seem to have this problem, although no specific
+details were reported so far. See the answer to the previous question, and
+please report if that helped.
 
 =head3 My Compose (Multi_key) key is no longer working.
 
@@ -914,25 +911,6 @@
 
 to your C<.profile> or C<.bashrc>.
 
-=head3 Why doesn't vim/emacs etc. use the 88 colour mode?
-
-See next entry.
-
-=head3 Why doesn't vim/emacs etc. make use of italic?
-
-See next entry.
-
-=head3 Why are the secondary screen-related options not working properly?
-
-Make sure you are using C<TERM=rxvt-unicode>. Some pre-packaged
-distributions (most notably Debian GNU/Linux) break rxvt-unicode
-by setting C<TERM> to C<rxvt>, which doesn't have these extra
-features. Unfortunately, some of these (most notably, again, Debian
-GNU/Linux) furthermore fail to even install the C<rxvt-unicode> terminfo
-file, so you will need to install it on your own (See the question B<When
-I log-in to another system it tells me about missing terminfo data?> on
-how to do this).
-
 
 =head2 Encoding / Locale / Input Method Issues
 
@@ -1089,14 +1067,12 @@
 
 =head3 I am using Debian GNU/Linux and have a problem...
 
-The Debian GNU/Linux package of rxvt-unicode in sarge contains large
-patches that considerably change the behaviour of rxvt-unicode (but
-unfortunately this notice has been removed). Before reporting a bug to
-the original rxvt-unicode author please download and install the genuine
-version (L<http://software.schmorp.de/pkg/rxvt-unicode.html>) and try to
-reproduce the problem. If you cannot, chances are that the problems are
-specific to Debian GNU/Linux, in which case it should be reported via the
-Debian Bug Tracking System (use C<reportbug> to report the bug).
+Before reporting a bug to the original rxvt-unicode author please download and
+install the genuine version
+(L<http://software.schmorp.de/pkg/rxvt-unicode.html>) and try to reproduce the
+problem. If you cannot, chances are that the problems are specific to Debian
+GNU/Linux, in which case it should be reported via the Debian Bug Tracking
+System (use C<reportbug> to report the bug).
 
 For other problems that also affect the Debian package, you can and
 probably should use the Debian BTS, too, because, after all, it's also a
