Description: Don't use hyphens as minus signs
Forwarded: no
Author: Ryan Kavanagh <ryanakca@kubuntu.org>
Origin: vendor
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Updated: 2011-03-11

Index: rxvt-unicode/doc/rxvt.7.pod
===================================================================
--- rxvt-unicode.orig/doc/rxvt.7.pod	2013-12-19 10:26:23.739221197 -0500
+++ rxvt-unicode/doc/rxvt.7.pod	2013-12-19 10:26:23.735221089 -0500
@@ -1974,7 +1974,7 @@
 
 =begin table
 
-	B<< C<h> >>	Bold/italic implies high intensity (see option B<-is>)
+	B<< C<h> >>	Bold/italic implies high intensity (see option B<\-is>)
 	B<< C<l> >>	Font styles have no effect on intensity (Compile styles)
 
 =end table
Index: rxvt-unicode/doc/rxvtc.1.pod
===================================================================
--- rxvt-unicode.orig/doc/rxvtc.1.pod	2013-12-19 10:26:23.739221197 -0500
+++ rxvt-unicode/doc/rxvtc.1.pod	2013-12-19 10:26:23.735221089 -0500
@@ -34,8 +34,8 @@
 
    #!/bin/sh
    @@RXVT_NAME@@c "$@"
-   if [ $? -eq 2 ]; then
-      @@RXVT_NAME@@d -q -o -f
+   if [ $? \-eq 2 ]; then
+      @@RXVT_NAME@@d \-q \-o \-f
       @@RXVT_NAME@@c "$@"
    fi
 
